core/dbt/config/runtime.py

```diff
@@ -98,6 +101,7 @@ class RuntimeConfig(Project, Profile, AdapterRequiredConfig):
     profile_name: str
     cli_vars: Dict[str, Any]
     dependencies: Optional[Mapping[str, "RuntimeConfig"]] = None
+    invocated_at: datetime = field(default_factory=lambda: datetime.now(pytz.UTC))
```

Comment:
Can we simplify this to `invoked_at`?

Reply:
We can 🙂 I went back and for on this rename myself. Will get it changed.